microreactors/gcmr/core/Serpent_Model/serpent_input.i

```diff
@@ -0,0 +1,637 @@
+64155.01c   1.76435E-03 
+64156.01c   2.42465E-03 
+64157.01c   1.84189E-03 
+64158.01c   2.90497E-03 
+64160.01c   2.52445E-03 
+8016.01c   3.52386E-02 
+
+mat Gd_bpO2 -7.0395 burn 1  tmp 725.0  rgb 0 0 200
+64152.01c   2.43139E-05 
+64154.01c   2.61576E-04 
+64155.01c   1.76435E-03 
+64156.01c   2.42465E-03 
+64157.01c   1.84189E-03 
+64158.01c   2.90497E-03 
+64160.01c   2.52445E-03 
+8016.01c   3.52386E-02 
+
+mat Gd_bpO3 -7.0395 burn 1  tmp 725.0  rgb 0 0 250
+64152.01c   2.43139E-05 
+64154.01c   2.61576E-04 
+64155.01c   1.76435E-03 
+64156.01c   2.42465E-03 
+64157.01c   1.84189E-03 
+64158.01c   2.90497E-03 
+64160.01c   2.52445E-03 
+8016.01c   3.52386E-02 
+
+mat boron_ctr -2.4700 tmp 725.0  rgb 0 0 250 % highly enriched B4C
+ 5010.01c 0.768
+ 5011.01c 0.032
+ 6012.01c 0.20
+
+mat coolant    -3.6500e-03  tmp 725.0 rgb 0 0 100                
+ 2004.01c  1.0  
+
+mat shell_mod  -7.0550  tmp 725.0   rgb 33 233 133  % density at 900K - FeCrAl
+24050.01c  8.6800E-03
+24052.01c  1.6762E-01
+24053.01c  1.8980E-02
+24054.01c  4.7200E-03
+26054.01c  4.3790E-02
+26056.01c  6.9249E-01
+26057.01c  1.6610E-02
+26058.01c  2.1140E-03
+13027.01c  4.5000E-02
+
+% --- Carbon buffer layer:
+
+mat buffer  -1.0400 moder grph 6012 tmp 725.0  rgb 0 200 200   
+ 6012.01c    1.0
+
+% --- Pyrolytic carbon layer:
+
+mat PyC1     -1.8820 moder grph 6012 tmp 725.0   rgb 0 200 200   
+ 6012.01c    1.0
+
+% --- Pyrolytic carbon layer:
+
+mat PyC2     -1.8820 moder grph 6012 tmp 725.0 rgb 0 200 200     
+ 6012.01c    1.0
+
+% --- Silicon carbide layer:
+
+mat SiC     -3.1710  tmp 725.0 rgb 0 100 0  
+14028.01c    0.4611
+14029.01c    0.0234
+14030.01c    0.0154
+ 6012.01c     0.5
+
+mat Nb   -8.4132  tmp 725.0   rgb 100 0 100   % Nobium 
+41093.01c  -0.99
+40090.01c  -5.132E-03
+40091.01c  -1.107E-03
+40092.01c  -1.674E-03
+40094.01c  -1.660E-03
+40096.01c  -2.618E-04
+
+mat Cr   -7.1900  tmp 725.0   rgb 100 0 200   % Chromium 
+24050.01c  -4.345e-2
+24052.01c  -83.789e-2
+24053.01c  -9.501e-2
+24054.01c  -2.365e-2
+
+% --- Graphite matrix:
+
+mat matrix_pin       -1.8060 moder grph 6012  tmp 725.0  rgb 220 220 220 
+ 6012.01c    1
+
+mat matrix       -1.8060 moder grph 6012  tmp 725.0 rgb 200 200 200      
+ 6012.01c  0.9999997
+ 5010.01c  0.0000003
+
+mat moderator   -4.0850 moder h-yh2 1001 moder y-yh2 39089 tmp 725.0   rgb 73 64 171
+39089.01c  0.357142857
+1001.01c   0.642857143
+
+mat Refl        -2.7987  moder beo 4009 tmp 725.0   rgb 100 220 200  % GREY
+ 4009.01c    0.5
+ 8016.01c    0.5
+
+% -----------------------------------------------------------------------------
+
+therm grph    grph.84t
+therm  beo     be-beo.84t
+therm h-yh2   h-yh2.84t
+therm y-yh2   y-yh2.84t
+
+% Calculation parameters:
+
+% -----------------------------------------------------------------------------
+
+% --- Geometry plot:
+plot 3 10000 10000   120.0
+plot 2 10000 10000   0.2
+
+% --- Libraries:
+
+set acelib "/home/ycao/XSLIB/serpent/sss_endfb8_serpent_yan.xsdir"
```

Comment:
add a comment to insert your own

how about you specify the paths on the INL HPC infrastructure? see the SNAP8 model serpent input